about.html

```diff
@@ -0,0 +1,163 @@
+                     and Africans that will enable them land high quality jobs worldwide.
+                </p>
+            </div>
+            <div class="about-headline-para-2">
+                <h1>
+                    We Provide FullStack <br> Web Development <br> Training
+                </h1>
+            </div>
+        </div>
+        <div class="content-first content-second">
+            <div class="reviews-title">
+                <h4>Why study with us?</h4>
+                <div class="underline"></div>
+            </div>
+            <div>
+                <p class="mission-text">
+                    We are set on a mission to provide highly paid in-demand IT skills to cameroonians
+                     and Africans that will enable them land high quality jobs worldwide.
+                </p>
+            </div>
+            <div class="about-cards-wrap">
+                <article class="about-card">
+                    <img src="images/expert-team.jpeg" alt="">
+                    <h3>EXPERT TEAM</h3>
+                    <p>we are a team driven by our vision to be the preferred web development training center in Cameroon.</p>
+                </article>
+                <article class="about-card">
+                    <img src="images/learn-code.jpeg" alt="">
+                    <h3>KNOW HOW TO CODE</h3>
+                    <p>Our objective is making sure you know how to code and get a job in the tech industry.</p>
+                </article>
+            </div>
+        </div>
+    </div>
+</section>
+
+ 
+<!-- PARTNERS -->
+<section class="partners">
+    <div class="reviews-title partners-title">
+        <h4> Our partners</h4>
+        <div class="underline"></div>
+    </div>
+    <div class="partners-wrap">
+        <i class="fab fa-google fa-3x"></i>
+        <i class="fab fa-microsoft fa-3x"></i>
+        <i class="fab fa-airbnb fa-3x"></i>
+        <i class="fab fa-amazon fa-3x"></i>
+    </div>
+</section>
+
+<!-- FOOTER -->
+<footer>
+    <div class="footer-inner">
+            <a href="index.html">
+                <div class="logo" id="footer-logo">
+                    <h3>My Logo</h3>
+                </div>
+            </a>
+        <div class="copy-right">
+            <div class="copy-right-mobile">
+                <p>2023 emmacroy Technologies</p>
+                <p>all rights reserved</p>
+            </div>
+            <div class="copy-right-desktop">
+                <p>2023 emmacroy Technologies. all rights reserved</p>
+                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Officia, possimus Officia, possimus. <br>
+                    consectetur adipisicing elit. Officia, possimus Officia, possimus
+                </p>
+            </div>
+        </div>
+    </div>
+     <!-- BACK TO TOP -->
+     <button class="back-to-top  ">
+        <a class="btn" href="#header" >
+          <i class="fas fa-arrow-up"></i>
+        </a>
+      </button>
+</footer>
```

Comment:
- [ ] Kindly hide the 'Partners' section of the desktop version of the 'About' page and change the colors of the footer to contrast with the section above it and match the template. The template has a dark background color and white text and logo.
 - Your design:
 - 
<img width="1086" alt="image" src="https://user-images.githubusercontent.com/61919617/217856134-a399f590-8c02-4d3b-aa36-f0944798dfd7.png">

 - Template:
 - 
<img width="887" alt="image" src="https://user-images.githubusercontent.com/61919617/217856213-5784e210-5e5b-4e25-bf44-e42c83a55624.png">